actions/contacts/create.go

```diff
@@ -23,13 +25,19 @@ import (
 // @Security	bux-auth-xpub
 func (a *Action) create(c *gin.Context) {
 
-	fullName := c.GetString("full_name")
-	paymail := c.GetString("paymail")
-	pubKey := c.GetString("pubKey")
+	var contactStruct engine.Contact
+
+	err := c.ShouldBindJSON(&contactStruct)
+
+	if err != nil {
+		fmt.Println("err", err)
+	}
```

Comment:
I think, fmt.Println should be replaced to use a logger or maybe the printing is not necessary here?

Reply:
Changed